<launch>
    <node pkg="sensor_synchronizer" type="synchronizer_node1" name="synchronizer" output="screen">
        <param name="recording_image_second" value="0.001" type="double"/>
        <param name="recording_image_frame" value="2" type="int"/>
        <param name="use_recording_image_second" value="true" type="bool"/>
        <param name="storage_mode" value="true" type="bool" />
        <param name="use_decawave_ranging" value="false" type="bool" />
        <!--<param name="direct" command="$(find filter_synchronizer1)/create_directory.sh"/>-->
    </node>
</launch>